test/cxx/ServerKit/HttpServerTest.cpp

```diff
@@ -687,7 +687,7 @@ namespace tut {
 			"Host: foo\r\n\r\n");
 		string response = readAll(fd, 1024).first;
 		ensure(containsSubstring(response,
-			"HTTP/1.1 505 HTTP Version Not Supported\r\n"
+			"HTTP/1.0 505 HTTP Version Not Supported\r\n"
```

Comment:
Why do you change to 1.0 here?

Reply:
Because the new parser picks up this error and returns its own response, which is an `http/1.0` response not `1.1`. Before we were catching the version mismatch and returning the error from our code with a `1.1` response.